src/connectors/star-rating/connectStarRating.js

```diff
@@ -196,8 +196,48 @@ export default function connectStarRating(renderFn, unmountFn) {
         return nextState;
       },
 
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`